python/TestHarness/testers/RunApp.py

```diff
@@ -200,6 +200,15 @@ def getPBSCommand(self, options):
     # Convert TEST_NAME to input tests file name (normally just 'tests')
     self.specs['no_copy'] = options.input_file_name
 
+    # Are we using the PBS Emulator, so make this param valid.
+    # The cluster_launcher will fill in the rest
+    if options.PBSEmulator:
+      self.specs['pbs_stdout'] = 'PBS_EMULATOR'
+      self.specs['pbs_stderr'] = 'PBS_EMULATOR'
+    else:
```

Comment:
Just get rid of this `else` clause. We don't need to populate empty parameters. The ClusterLauncher will do the right thing when those parameters are just not supplied.